ethereum/spec/src/main/java/tech/pegasys/teku/spec/datastructures/blocks/blockbody/versions/altair/BeaconBlockUnblinderAltair.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2022 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.spec.datastructures.blocks.blockbody.versions.altair;
+
+import tech.pegasys.teku.spec.datastructures.blocks.SignedBeaconBlock;
+import tech.pegasys.teku.spec.datastructures.blocks.blockbody.versions.phase0.BeaconBlockUnblinderPhase0;
+import tech.pegasys.teku.spec.schemas.SchemaDefinitions;
+
+public class BeaconBlockUnblinderAltair extends BeaconBlockUnblinderPhase0 {
```

Comment:
I'd be tempted to just use the `Phase0` version for Altair as well and not bother with an Altair specific class.  There's quite a few things that Altair inherited unchanged from phase0 already.